apps/include/netutils/libcoap/address.h

```diff
@@ -0,0 +1,164 @@
+/* address.h -- representation of network addresses
```

Comment:
Can you explain why we need libcoap? To add a new feature(or lib), we need to discuss the requirement of the feature together.